Translate examples in English (#104)
ambanum/disinfo.quaidorsay.fr: _pages/en/encyclopedia/examples.md

---
title: Examples of information manipulation campaigns
permalink: /encyclopedia/examples
nav_order: 3
layout: default

> nav_exclude: true

Reviewer: We probably want to have this page appear in the navigation menu 🙂

---

# Examples of information manipulation campaigns

This page references recent, important and documented campaigns to illustrate the diversity of information manipulation, in terms of sources, channels used and goals pursued.

## Sources

Information manipulation can be orchestrated by a variety of sources: state or non-state actors, such as companies or organisations, or even informal groups of individuals.
These sources may position themselves in different ways in relation to the audience targeted by the information manipulation. Thus, in the case of what can be described as "exogenous" manipulation, the source is outside the target community, or does not depend directly on it.
Conversely, in a manipulation of information that can be described as "endogenous", the source belongs to the target community, the target country or is highly dependent on it.

> When the manipulation of information is of state and foreign source, it is an [interference](/encyclopedia/glossary).

Reviewer: ```suggestion
When the manipulation of information originates from a foreign state source, it is an [interference](/encyclopedia/glossary#interference).
```

> Nevertheless, it must be stressed that one of the greatest difficulties in the fight against information manipulation remains the question of [attribution](/encyclopedia/attribution). Tracing the identity of the source and attributing responsibility is always complex, and certainty is almost impossible to achieve. The exercise is made all the more delicate by the fact that third parties may try to give false clues to guide attribution. For this reason, all the examples of campaigns given below only list the sources suspected by the authors of the analyses, and do not represent attribution.

Reviewer: ```suggestion
One of the greatest difficulties in countering information manipulation is [attributing](/encyclopedia/attribution) campaigns. Tracing the identity of the source and attributing responsibility is always complex, and certainty is almost impossible to achieve. This is made even harder by the fact that third parties may give false clues to mislead attribution. For this reason, all the examples of campaigns given below only list the sources suspected by the authors of the analyses, and do not represent official attribution.
```

| The Macron Leaks |
|-|
| **Date:** 2017|

> | **Nature of the suspected actors:** state actor (Russia), American alt-right, French fachosphere|

Reviewer: ```suggestion
| **Nature of the suspected actors:** state actor (Russia), American alt-right, French _fachosphère_ (extreme right-wing digital actors) |
```

> | **Type of manipulation:** exogenous|

Reviewer: Does this really characterize the _type of manipulation_? 🤔

| **Channels used:**  information sites relaying pro-Kremlin narratives (Sputnik, RT), social networks|

> | **Modus operandi:** use of trolls, cyberattackers, and leak of a mix of fake and real documents, distributed using artificial amplification on social networks|

Reviewer: ```suggestion
| **Modus operandi:** use of trolls, cyberattacks, leak, forgery, artificial amplification on social networks|
```

> | **Supposed goals:** weaken the Macron candidate, sow doubt about the election result|

Reviewer: ```suggestion
| **Supposed goals:** weaken then-candidate Macron, sow doubt about the election result|
```

| **Background:** 2017 French presidential elections|

> | **Summary:** The French presidential campaign was the theatre of the Macron Leaks. For months a campaign of information manipulation was orchestrated on social networks and on certain media (Sputnik and Russia Today in particular) against the Macron candidate. This campaign was completed two days before the second round of the presidential election, and more precisely two hours before the last televised debate, by a leak of false documents allegedly hacked from the computers of the candidate's campaign team, a leak which was later artificially amplified on social networks. Russia and extreme right-wing American and French groups are suspected of being involved in this manipulation.|

Reviewer: ```suggestion
| **Summary:** The French presidential campaign was the theatre of the Macron Leaks. For months a campaign of information manipulation was orchestrated on social networks and on certain media (Sputnik and Russia Today in particular) against candidate Macron. This campaign was completed two days before the second round of the presidential election, and more precisely two hours before the last televised debate, by a leak of documents allegedly hacked from the computers of the candidate's campaign team, a leak which was later artificially amplified on social networks. Russia and extreme right-wing American and French groups are suspected of being involved in this manipulation.|
```

| **Source :** [CAPS](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/) |

| Go bald for BLM|
|-|
| **Date:** 2020|
| **Nature of the suspected actors:** civil society|
| **Type of manipulation:** endogenous|
| **Channels used:** social networks, 4Chan forum|
| **Modus operandi:** launching a false rumor, organic and coordinated amplification|

> | **Supposed goals:** to discredit the Black Lives Matter movement|

Reviewer: ```suggestion
| **Supposed goals:** discredit the Black Lives Matter movement|
```

| **Background:** Black Lives Matter movement of protest against systemic racism|

> | **Summary:** In November 2020, users of the American forum 4chan, decide to launch a fake challenge on social networks: ask white women to shave their heads as a sign of support for the Black Lives Matter movement, in order to discredit the movement. 4chan users (quickly relayed by other communities) amplify the challenge by posting fake photos of women with shaved heads and associating it with the hashtag #GoBaldForBLM.|

Reviewer: I guess the photos themselves aren't _fake_, but what is fake is the cause of the baldness.
```suggestion
| **Summary:** In November 2020, some users of the American forum 4chan decide to launch a fake challenge on social networks: ask white women to shave their heads as a sign of support for the Black Lives Matter movement, in order to discredit the movement. These users, quickly joined by other communities, amplify the challenge by posting photos of women with shaved heads and associating it with the hashtag #GoBaldForBLM.|
```

| **Source :** [France24](https://observers.france24.com/en/20200611-debunked-4chan-discredit-black-lives-matter-bald-women-fail)|

> ## Channels ##

Reviewer: ```suggestion
## Channels
```

> The manipulation of information can spread through several channels: media (television, radio, newspapers), social networks, particularly through the use of fake accounts or sponsored content.

Reviewer: ```suggestion
Information manipulation can be spread through several channels: media (television, radio, newspapers), social networks, particularly through the use of fake accounts or sponsored content.
```

These channels are used with different _modus operandi_: information retention, censorship, artificial amplification of information, collection of personal data, targeted campaigns, etc.

> The protean nature of information manipulations and the use of various channels and _modus operandi_ in conjunction can complicate the characterization effort.

Reviewer: ```suggestion
The combined variety of information manipulation forms, channels and _modus operandi_ makes characterization all the more difficult.
```

| The U.S. presidential elections of 2016|
|-|
| **Date:** 2016|

> | **Nature of the suspected actors:** state actor (GRU), private company close to state actors, Internet Research Agency (IRA), private company (Cambridge Analytica)|

Reviewer: ```suggestion
| **Nature of the suspected actors:** state actor (GRU, Russia), private company close to state actors (Internet Research Agency), private company (Cambridge Analytica)|
```

| **Type of manipulation:**  exogenous for GRU and IRA, endogenous for Cambridge Analytica|
| **Channels used:** social networks, forums, websites|
| **Modus operandi:** cyberattacks and leak of sensitive documents, targeted campaign by collecting personal data, use of troll farms|
| **Supposed goals:** to destabilize and divide American society, to promote the election of a candidate|
| **Background:** 2016 US presidential elections |

> | **Summary:**  The US presidential campaign is marked by cyber attacks and manipulation of information, some of which are identified as Russian interference by the United States.<br>• WikiLeaks publishes 30,000 emails revealing certain exchanges between candidate Clinton and her team when she was Secretary of State. The controversy surrounding Hillary Clinton's use of a personal email address for confidential conversations is revived. The Mueller report will attribute the exfiltration of these documents to the GRU (Russian military intelligence service).<br>• Cambridge Analytica, a British company headed by Steve Bannon, has been hired by the Trump candidate's team to conduct a media campaign on Facebook. The company is recovering the personal data of 87 million people via a third party application, and is running a targeted campaign in the swing states against individuals selected on the basis of their personal data.<br>• The IRA is suspected of using troll farms to influence public opinion on social networks. According to Facebook, nearly $100,000 worth of targeted ads were purchased, and thousands of bots and fake accounts have allegedly amplified this false information (but also real information such as the leak of Clinton's emails).|

Reviewer: ```suggestion
| **Summary:**  The US presidential campaign is marked by cyber attacks and manipulations of information, some of which are identified as Russian interference by the United States.<br>• WikiLeaks publishes 30,000 emails revealing certain exchanges between candidate Clinton and her team when she was Secretary of State. The controversy surrounding Hillary Clinton's use of a personal email address for confidential conversations is revived. The Mueller report will attribute the exfiltration of these documents to the GRU (Russian military intelligence service).<br>• Cambridge Analytica, a British company headed by Steve Bannon, has been hired by the Trump candidate's team to conduct a media campaign on Facebook. The company is recovering the personal data of 87 million people via a third party application, and is running a targeted campaign in the swing states against individuals selected on the basis of their personal data.<br>• The IRA is suspected of using troll farms to influence public opinion on social networks. According to Facebook, nearly $100,000 worth of targeted ads were purchased, and thousands of bots and fake accounts have allegedly amplified this false information, as well as real information such as the leak of Clinton's emails.|
```

| **Source :** [France24](https://www.france24.com/fr/20170907-internet-research-agency-trolls-russes-facebook-achat-publicite-propagande-trump-election), [BusinessInsider's summary](https://www.businessinsider.fr/us/cambridge-analytica-a-guide-to-the-trump-linked-data-firm-that-harvested-50-million-facebook-profiles-2018-3)|

| The Brazilian presidential elections of 2018|
|-|
| **Date:** 2018|
| **Nature of the suspected actors:** political party, communication companies|
| **Type of manipulation:** endogenous|
| **Channels used:** messaging application (WhatsApp)|
| **Modus operandi:** propagation of fake news, mass communication to  citizens' personal phones|
| **Supposed goals:** to favour the candidate Jair Bolsonaro and denigrate his opponents|
| **Background:** 2018 Brazilian presidential elections |
| **Summary:** The Brazilian presidential elections are won by the far-right candidate Jair Bolsonaro. The campaign is marked by the use of WhatsApp to propagate messages with political content, which proved to be fake news in most cases. Bolsonaro's party has been investigated for organizing a disinformation campaign on WhatsApp using mass media companies.|

> | **Source :** [The Guardian](https://www.theguardian.com/world/2019/oct/30/whatsapp-fake-news-brazil-election-favoured-jair-bolsonaro-analysis-suggests), [Folha de S. Paulo](https://www1.folha.uol.com.br/internacional/en/brazil/2018/10/businessmen-fund-whatsapp-campaign-against-pt.shtml) |

Reviewer: Remove space (repeated for all `Source` items).
```suggestion
| **Source:** [The Guardian](https://www.theguardian.com/world/2019/oct/30/whatsapp-fake-news-brazil-election-favoured-jair-bolsonaro-analysis-suggests), [Folha de S. Paulo](https://www1.folha.uol.com.br/internacional/en/brazil/2018/10/businessmen-fund-whatsapp-campaign-against-pt.shtml) |
```

## Goals

> Information manipulation can support a wide variety of goals: improving the image of the attacker, hushing up a case, destabilising a personality or community, exacerbating pre-existing tensions, dividing and sowing chaos, undermining confidence in institutions, altering the sincerity of the ballot.

Reviewer: ```suggestion
Information manipulation can support a wide variety of goals: improving the image of the attacker, hushing up a case, destabilising a personality or community, exacerbating pre-existing tensions, dividing society, undermining confidence in institutions, undermining elections integrity…
```

| **Type of manipulation:** endogenous|
| **Channels used:** social networks|
| **Modus operandi:** use of fake accounts to amplify false information|
| **Supposed goals:** to incite hatred towards the Rohingyas|
| **Background:** Violence against the Rohingya ethnic group in Myanmar|
| **Summary:** The Rohingya crisis is at its peak in Myanmar. Facebook reveals that the military junta is conducting coordinated information manipulation campaigns using hundreds of fake accounts to incite hatred towards the Rohingyas and thus justify their forced migration.|
| **Source :** [New York Times](https://www.nytimes.com/2018/10/15/technology/myanmar-facebook-genocide.html)|

| The 2019 UK general election|
|-|
| **Date:** 2019|
| **Nature of the suspected actors:** state actor (Russia)|
| **Type of manipulation:** exogenous |
| **Channels used:** social networks at first, then media naturally amplified|
| **Modus operandi:** cyber attack then leak of an authentic document, artificial amplification on social networks|
| **Supposed goals:** change the outcome of an election, sow doubt, undermine confidence in government|
| **Background:** UK general election of 2019|

> | **Summary:**  In the United Kingdom, the parliamentary elections are marked by the leak on Reddit of a draft trade agreement with the United States. The documents were found to be genuine. Reddit removed 61 accounts that coordinated the amplification of information. The leader of the opposition, Jeremy Corbyn, considered this agreement to be a weakening of the British health care system and used these documents to attack the government during a press conference. In June 2020, the American company Graphika, which specialises in the study of the manipulation of information on the Internet, linked this event to a larger operation nicknamed Secondary Infektion. In July 2020, the British government officially declared Russia responsible for the cyber attack, the leak and the artificial amplification of the document.|

Reviewer: ```suggestion
| **Summary:**  In the United Kingdom, the parliamentary elections are marked by the leak on Reddit of a draft trade agreement with the United States. The documents were found to be genuine. Reddit removed 61 accounts that coordinated the amplification of information. The leader of the opposition, Jeremy Corbyn, considered this agreement to be a weakening of the British health care system and used these documents to attack the government during a press conference. In June 2020 American company Graphika, a specialist of the study of digital information manipulation, linked this event to a larger operation nicknamed Secondary Infektion. In July 2020, the British government officially declared Russia responsible for the cyber attack, the leak and the artificial amplification of the document.|
```
